sql/2022/fonts/hosted_vs_self_hosted.sql

```diff
@@ -0,0 +1,38 @@
+#standardSQL #self_hosted_vs_hosted
+SELECT
+  client,
+  CASE
+    WHEN pct_self_hosted_hosted = 1 THEN 'self-hosted'
+    WHEN pct_self_hosted_hosted = 0 THEN 'external'
+    ELSE 'both'
+  END AS font_host,
+  COUNT(DISTINCT page) AS pages,
+  SUM(COUNT(DISTINCT page)) OVER (PARTITION BY client) AS total,
```

Comment:
Similar comment about the total
Not sure of the purpose of the join

Reply:
This file was copied over from 2020. It is replaced by the usage by services query, so I ended up just deleting this one.